nexuse2e-webapp/src/main/frontend/src/app/services/data.service.ts

```diff
@@ -130,6 +128,26 @@ export class DataService {
     );
   }
 
+  getEngineLogs(
+    pageIndex: number,
+    itemsPerPage: number,
+    activeFilters: ActiveFilterList
+  ): Promise<EngineLog[]> {
+    let params = DataService.buildFilterParams(activeFilters)
+    ;
```

Comment:
```suggestion
    let params = DataService.buildFilterParams(activeFilters);
```

Reply:
Formatting was broken, so now it is done.